README.md

```diff
@@ -121,8 +134,8 @@ This Tool returns `data` with following format
 | -------------- | --------- | ------------------------------- |
 | file           | `object`  | Uploaded file data. Any data got from backend uploader. Always contain the `url` property |
 | caption        | `string`  | image's caption                 |
-| withBorder     | `boolean` | add border to image             |
-| withBackground | `boolean` | need to add background          |
+| border         | `boolean` | add border to image             |
+| background     | `boolean` | need to add background          |
```

Comment:
this names should not be changed, previous names was ok and widely used

Reply:
only "features" should have short notation since there are features listed, not tunes or image properties